### PR TITLE
add logs when getting and putting storage

### PR DESCRIPTION
**Pull request by K-Ho**
## Questions
- 
-
-

### Fixes
- Fixes # [Link to Issue]

## Contributing Agreement

- [x] I have read and understood the [Optimism Contributing Guide and Code of Conduct](./CONTRIBUTING.md) and am following those guidelines in this pull request.

### CODE REVIEW

File: core/vm/ovm_state_manager.go [hunk 1]
```diff
@@ -111,6 +112,7 @@ func getContractStorage(evm *EVM, contract *Contract, args map[string]interface{
 	}
 	key := toHash(_key)
 	val := evm.StateDB.GetState(address, key)
+	log.Debug("Got contract storage:", "address", address.Hex(), "key", key.Hex(), "val", val.Hex())
```
Comment (tynes) [inline core/vm/ovm_state_manager.go]: No need for `:`

**Comment by tynes** [comment]
Maybe we could turn these into `log.Trace` so that we could turn them off by changing the `--verbosity` flag at runtime? Not sure if we would want to see these logs in production, what do you think?

**Comment by K-Ho** [comment]
> Maybe we could turn these into `log.Trace` so that we could turn them off by changing the `--verbosity` flag at runtime? Not sure if we would want to see these logs in production, what do you think?

I think they'd be really helpful in production! Very hard to debug with our logs right now without knowing the values that we're SSTOREing and SLOADing

**Review by karlfloersch — APPROVED** [review]
LGTM!

**Pull request merged**